### PR TITLE
Fix share link

### LINKED ISSUE #207: Share link bugs
- Firefox incognito mode (private move)
- When user user login with deep link the share link is wrong (it use web wallet instead of superhero.com)

### LINKED ISSUE #209: Login in Brave browser or privat mode
Hello team! 

I have tested Jitsi Meet in Brave browser. After logging in with the iframe wallet, I entered the meeting room, but the dashboard items are not visible or active (the room is grey).

Знімок екрана 2020-08-28 о 12 14 17 пп

The same issue in private mode in Firefox with or without wallet login. The conference room is not available and has grey view:

Знімок екрана 2020-08-28 о 3 45 07 пп

Знімок екрана 2020-08-28 о 1 34 01 пп

Знімок екрана 2020-08-28 о 1 34 14 пп

### LINKED ISSUE #211: Invite link doesn't work correctly
When I log into Jitsi Meet (I use the web wallet) invite link does not work correctly: https://wallet.superhero.com/sign-message?x-success=https%3A%2F%2Fmeet.aepps.com%2FGladDaysRemindOnce%3Fresult%3Dsuccess%26signature%3D%7Bsignature%7D&x-cancel=https%3A%2F%2Fmeet.aepps.com%2FGladDaysRemindOnce%3Faddress%3Dak_cEjh5on5RvoBw8zqCGS4VWeY2bmNYNPjn3SCZutYyVLWKf7X1&message=I%20would%20like%20to%20generate%20JWT%20token%20at%20Fri,%2028%20Aug%202020%2012%3A30%3A56%20GMT

Знімок екрана 2020-08-28 о 1 16 35 пп

### PR DESCRIPTION
**Pull request by shapkarin**
issues: 
https://github.com/aeternity/jitsi-meet/issues/211
https://github.com/aeternity/jitsi-meet/issues/209
also closes #207

more info: https://github.com/aeternity/jitsi-meet/issues/111#issuecomment-675743340

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -5,6 +5,10 @@ var config = {
     //
 
     hosts: {
+
+        // parent document jitsi path
+        parent: 'https://superhero.com/meet',
```
Comment (Chima1707) [inline config.js:10]: What happens when we have superhero, stage, dev, prod? Do they still use same parent host?

File: react/features/base/connection/functions.js [hunk 2]
```diff
@@ -41,7 +43,9 @@ export function getCurrentConferenceUrl(stateful: Function | Object) {
  */
 export function getInviteURL(stateOrGetState: Function | Object): string {
     if (isInIframe()) {
```
Comment (venimus) [inline react/features/base/connection/functions.js]:
```suggestion
    if (isInIframe() && config.hosts.parent) {
```

**Review by Chima1707 — CHANGES_REQUESTED** [review]

File: config.js (hunk 1 above)
Comment (shapkarin) [reply config.js:10]: It's different instances that use different config. @venimus can confirm that.

Comment (shapkarin) [reply config.js:10]: @Chima1707 not sure how to track that in repo.

Comment (Chima1707) [reply config.js:10]: @venimus Please can you suggest something here?

Comment (venimus) [reply config.js:10]: it is ok like this for now, will make changes per sever

**Review by venimus — APPROVED** [review]

File: config.js (hunk 1 above)
Comment (Chima1707) [reply config.js:10]: Ok, great!

**Comment by shapkarin** [comment]
@Chima1707 can you please disable the change request?

**Review by Chima1707 — APPROVED** [review]

**Pull request merged**